src/common/ApprovalsChip.tsx

```diff
@@ -1,14 +1,31 @@
+import React, { useState, MouseEvent } from 'react'
+
 import Close from '@mui/icons-material/CloseTwoTone'
-import DoneAll from '@mui/icons-material/DoneAllTwoTone'
 import Done from '@mui/icons-material/DoneTwoTone'
+import DoneAll from '@mui/icons-material/DoneAllTwoTone'
 import Chip from '@mui/material/Chip'
+import Menu from '@mui/material/Menu'
+import List from '@mui/material/List'
+import ListItem from '@mui/material/ListItem'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import AccessTime from '@mui/icons-material/AccessTime'
+import DownArrow from '@mui/icons-material/KeyboardArrowDown'
+import UpArrow from '@mui/icons-material/KeyboardArrowUp'
+import Stack from '@mui/material/Stack'
+
 import useTheme from '@mui/styles/useTheme'
-import React from 'react'
-import { Theme } from '../theme'
+import { Theme } from '../../src/theme'
 
 export default function ApprovalsChip({ approvals }: { approvals: any }) {
```

Comment:
Having props typed as `any` makes things harder than they need to be. Does it make sense to swap out `approvals` for `managerResponse` and `reviewerResponse` as two strongly typed objects? As it stands there's a lot of properties blindly being accessed.